src/index.js

```diff
@@ -94,6 +94,70 @@ class VersionHash {
             }
 
         })
+
+        // files inside css
+        forIn(webpackConfig.module.rules, rule => {
+            // check if the rule is /(\.(png|jpe?g|gif|webp)$|^((?!font).)*\.svg$)/
+            if ('.png'.match(new RegExp(rule.test))) {
```

Comment:
'.png', '.woff' and '.cur', they are just for testing the regular expression of the rule already exists in the original Laravel-mix, and thus make the appropriate changes.

about your second question. I just followed the rules the original Laravel-mix, svg and images are together, and as we are just modifying the "name" property, we must follow the Laravel-mix rules, otherwise it will break the code. Take a look at https://github.com/JeffreyWay/laravel-mix/blob/master/src/builder/webpack-rules.js, lines 13, 51 and 77.

Reply:
complementing...

"forIn" loops are about webpack rules, not project files. 

[filename].svg => /(\.(png|jpe?g|gif|webp)$|^((?!font).)*\.svg$)/ --> image
[filename].font.svg => /(\.(woff2?|ttf|eot|otf)$|font.*\.svg$)/ --> font

Reply:
oh ok, thanx for the headsup